crates/orchestrator/src/data_storage/aws_s3/mod.rs

```diff
@@ -0,0 +1,130 @@
+use crate::data_storage::aws_s3::config::AWSS3Config;
+use crate::data_storage::types::StarknetOsOutput;
+use crate::data_storage::DataStorage;
+use async_trait::async_trait;
+use aws_sdk_s3::config::{Builder, Credentials, Region};
+use aws_sdk_s3::primitives::ByteStream;
+use aws_sdk_s3::{Client, Error};
+
+pub mod config;
+
+const SNOS_OUTPUT_FILE_NAME: &str = "snos_output.json";
+const KZG_FILE_NAME: &str = "kzg.txt";
```

Comment:
s3 shouldn't be aware of this, these details can be in the snos job. also, do we need to store kzg now? i was thinking we can calculate the proof when we calculate state diffs in update state only